File: 1480/1480.running-sum-of-1-d-array.java
/*
 * @lc app=leetcode id=1480 lang=java
 *
 * [1480] Running Sum of 1d Array
 */

// @lc code=start
class Solution {
    public int[] runningSum(int[] nums) {
        int[] result = new int[nums.length];

        for (int k = 0; k < nums.length; k++){
            for (int l = 0; l <= k; l++){
                result[k] += nums[l];
            }
        }

        return result;
    }
}
// @lc code=end

File: 1480/1480.running-sum-of-1-d-array.py
#
# @lc app=leetcode id=1480 lang=python3
#
# [1480] Running Sum of 1d Array
#

# @lc code=start
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        result = []

        for index, value in enumerate(nums):
            result.append(0)
            result[index] = result[index-1] + value 

        return result
# @lc code=end

